### DISCUSSION TITLE
How I can generated a schema for nextjs in server side mode

### DISCUSSION THREAD

**Discussion by edw19** [MichalLytek/type-graphql#940] [Q&A]
createSchema call the function buildSchema() 

![image](https://user-images.githubusercontent.com/47750859/124004925-1c6e8f80-d99e-11eb-9608-b49ca4c5ac02.png)

but i have this error 

![image](https://user-images.githubusercontent.com/47750859/124005080-4922a700-d99e-11eb-9610-560876510347.png)

**Comment by MichalLytek** [comment]
Try using `buildSchemaSync` and removing the `async/await` wrapper.

**Reply by edw19** [reply]
That fixes the schema problem, thanks but I'm using @Authorized in my resolvers and apparently at the time of generating this schema I don't have access to the req

![image](https://user-images.githubusercontent.com/47750859/124009178-e8499d80-d9a2-11eb-8da6-ac52eab54ea2.png)

**Reply by MichalLytek** [reply]
So you need to configure the context in schema link. Please refer to the apollo client and isomorphic schema link. It's not coupled to TypeGraphQL.
